django_girls.html

```diff
@@ -51,18 +50,19 @@
                 </section>
                 <section>
                     <section>
-                        <h1>Future</h1>
+                        <h1>Przyszłość</h1>
                     </section>
                             <section>
                                 <img src="img/wysadz_ulice.png" style="background:none; border:none; box-shadow:none; "/>
                             </section>
                     </section>
                 </section>
                 <section>
-                    <h1>Questions?</h1>
-                    <h5 class="fragment">Thank you for your attention</h2>
-                    <h5 class="fragment"><a href="mailto:wojciech.patelka@gmail.com">wojciech.patelka@gmail.com</a></h2>
+                    <h1>Pytania?</h1>
+                    <h5 class="fragment">Dziękuję za uwagę.</h2>
+                    <h5 class="fragment"><a href="mailto:wojciech.patelka@gmail.com">hello@codeforpoznan.pl</a></h2>
                     <h5 class="fragment"><a href="https://github.com/CodeForPoznan">https://github.com/CodeForPoznan</a></h3>
+                    <h5 class="fragment"><a href="www.codeforpoznan.pl">codeforpoznan.pl</a>
```

Comment:
if we use https in github address we should use http here also (in body and in href)